### ISSUE TITLE
Integrate with Digital Asset Management system for images

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#337]
**In order to server images, we will integrate with a Digital Asset Management system**

## Acceptance criteria
1. It is possible to browse and select images located in DAM from Sanity Studio

## In order to ensure good quality we should (Keep the ones we need for this US)
- [ ] Perform integration testing
- [ ] Perform functional testing
- [ ] Extra focus on usability
- [ ] Test for portability
- [ ] Test for accessibility
- [ ] Create automated acceptance test/smoke test
- [ ] Perform performance and load test
- [ ] Perform user testing
- [ ] Perform security testing

**Comment by wenche** [comment]
Since Equinor might be moving away from the current DAM solution at some point, we decided that we should not spend a lot of time on this. We'll reuse the POC plugin, but try to reduce the amount of hard coded value and lift it to using TypeScript.

**Comment by SvSven** [comment]
Implementing the brandmaster asset source is causing a problem with the build to fail:

```bash
Error in ./node_modules/@sanity/form-builder/lib/legacyParts.js
Module not found: Error: Can't resolve 'all:part:@sanity/form-builder/input/image/asset-source'
```

Published asset-source packages (like Cloudinary and Unsplash) work fine, but those are compiled with an older version of Sanity.
Waiting to hear back from Sanity about the issue before we can proceed.

**Comment by wenche** [comment]
Related conversation https://equinor.slack.com/archives/C01A4BGNRMJ/p1625487316003300

**Comment by SvSven** [comment]
Checked to see if the issue may have magically been solved by updated dependencies, but the problem persists.
Awaiting meeting with Sanity to discuss it sometime next week.

**Comment by nilsml** [comment]
What to do we do with meta data?

**Comment by nicholas-haggerty** [comment]
@meols take dialogue with Yngve.

**Comment by SvSven** [comment]
Waiting on #443

**Comment by nilsml** [comment]
Created new issue for the Brandmaster cookie issue #476
